src/systems/floaty-object-system.js

```diff
@@ -124,9 +124,9 @@ export const floatyObjectSystem = world => {
 
   networkedFloatyObjectsQuery(world).forEach(eid => {
     if (hasComponent(world, Owned, eid)) {
-      NetworkedFloatyObject.flags = FloatyObject.flags;
+      NetworkedFloatyObject.flags[eid] = FloatyObject.flags[eid];
```

Comment:
This is a js file, but like we talked about, would be nice if we can modify the BitECS typescript defs to catch this sort of mistake. I think we have made it more than once.